taginfo.json

```diff
@@ -2767,6 +2767,206 @@
       ],
       "description": "Allows access for all modes unless the mode override is set except for bicycles."
     },
+    {
+      "key": "construction",
+      "value": "motorway",
+      "object_types": [
+        "way"
+      ],
+      "description": "Motorway road under construction."
+    },
```

Comment:
added values based on https://wiki.openstreetmap.org/wiki/Tag:highway%3Dconstruction and values in `highway` tag from this file